### PR TITLE
Set Maven compiler and release versions to 7 for integration test projects

### PR DESCRIPTION
**Pull request by MartinKanters**
See discussion here: https://github.com/apache/maven-integration-testing/pull/135#discussion_r787104645.

### CODE REVIEW

**Review by michael-o — APPROVED** [review]
Shit, this should have never happened. Thank you for being vigilant!

**Review by mthmulders — APPROVED** [review]
```sh
$ grep -R "source>1.8" core-it-suite/src/test/resources | wc -l
9
```
I guess you had all occurrences :-)

**Comment by michael-o** [comment]
Check with >8 too

**Comment by MartinKanters** [comment]
@mthmulders Nice find! Fixed that as well.

@michael-o Checked > 8, but there are no occurrences as far as I can see.

File: core-it-suite/src/test/resources/mng-7045/pom.xml [hunk 1]
```diff
@@ -25,8 +25,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.8.1</version>
         <configuration>
-          <source>8</source>
-          <target>8</target>
+          <source>7</source>
```
Comment (mthmulders) [inline core-it-suite/src/test/resources/mng-7045/pom.xml:28]: Is that valid? I thought dropping the `1.` is allowed only from 8 onwards, but I'm not sure...

Comment (michael-o) [reply core-it-suite/src/test/resources/mng-7045/pom.xml:28]: Likely depends on the Java version used

Comment (MartinKanters) [reply core-it-suite/src/test/resources/mng-7045/pom.xml]: Good point, I was a bit too hasty there. For my JDK (OpenJDK) it worked with "7", but made it "1.7" just to be sure. It works with that as well.

**Pull request merged**